CONTRIBUTING.rst

```diff
@@ -203,16 +203,67 @@ Docstrings
 
 PyVista uses Python docstrings to create reference documentation for our Python
 APIs. Docstrings are read by developers, interactive Python users, and readers
-of our online documentation. This page describes how to write these docstrings
+of our online documentation. This section describes how to write these docstrings
 for PyVista.
 
-* PyVista follows the ``numpydoc`` style for its docstrings. Please follow the
-  `numpydoc Style Guide`_.
+PyVista follows the ``numpydoc`` style for its docstrings. Please follow the
+`numpydoc Style Guide`_ in all ways except for the following:
+
 * Be sure to describe all ``Parameters`` and ``Returns`` for all public
   methods.
 * We strongly encourage you to add an example section. PyVista is a visual
   library, so adding examples that show a plot will really help users figure
   out what individual methods do.
+* With optional parameters, use ``default: <value>`` instead of ``optional``
```

Comment:
In https://github.com/pyvista/pyvista/pull/3457/files we seemed to use `default=<value>`. Either works for me, we should just make sure we choose one on purpose.

Reply:
The current code base has more `default: <value>`.  I found 164 hits with this. 
 This includes some false positives, but not as many as I would have suspected.  I found 36 hits for `default=<value>`, with also some false positives.  Before #3457, it would have leaned even heavier towards the colon usage.  Although, I'm fine either way as well.

Reply:
I'd also prefer a colon here, ignoring all legacy considerations.

Reply:
Agree with the colon. Let's go with that and consider a refactor. At least it's low hanging.